### DISCUSSION TITLE
kornia in Random seeds

### DISCUSSION THREAD

**Discussion by 464hee** [kornia/kornia#2343] [General]
Whether to support setting random seeds in order to complete the recreation of good models.
If so, please let me know, as I can't seem to find a description in the documentation.

**Comment by edgarriba** [comment]
@464hee random generation only happens in `kornia.augmentations` and since we only depend on pytorch, with `torch.maual_seed` should be enough
*https://pytorch.org/docs/stable/generated/torch.manual_seed.html#torch.manual_seed

**Comment by 464hee** [comment]
Thanks for your reply, I took the following code so that each time I rerun it, the image enhancement results are consistent. It might be enough to just s set up pytorchj, but just to be safe, I set up the other two related ones the same way

seed = 2023
import random
random.seed(seed)
import numpy as np
np.random.seed(seed)
import torch
torch.manual_seed(seed)

**Comment by edgarriba** [comment]
makes sense yes
